### ISSUE TITLE
error: expected identifier, found `_`

### ISSUE THREAD

**Issue by ctaggart** [ctaggart/octh#12]
```
error: expected identifier, found `_`
   --> src\lib.rs:204:15
    |
204 |     pub const _: ::std::os::raw::c_uint = 1;
    |               ^
    |
    = note: `_` is a wildcard pattern, not an identifier
```

**Comment by ctaggart** [comment]
```
    pub const __MINGW64_VERSION_STATE: &'static [u8; 7usize] = b"stable\x00";
    pub const __MINGW32_MAJOR_VERSION: ::std::os::raw::c_uint = 3;
    pub const __MINGW32_MINOR_VERSION: ::std::os::raw::c_uint = 11;
    pub const _: ::std::os::raw::c_uint = 1;
    pub const __MINGW_USE_UNDERSCORE_PREFIX: ::std::os::raw::c_uint = 0;
    pub const _M_AMD64: ::std::os::raw::c_uint = 100;
    pub const _M_X64: ::std::os::raw::c_uint = 100;
```

Looks to be generated from C:\Octave\Octave-4.2.1\include\_mingw_mac.h

**Comment by ctaggart** [comment]
I'm able to reproduce the problem with just this line:
https://github.com/ctaggart/mingw-w64/blob/master/mingw-w64-headers/crt/_mingw_mac.h#L90
``` h
#      define _ 1
```
If we look at the comments, it looks like this is for supporting older gcc where older versions are using an underscore prefix.
https://github.com/ctaggart/mingw-w64/blob/master/mingw-w64-headers/crt/_mingw_mac.h#L81-L106
``` h
#ifndef _X86_
   /* MS does not prefix symbols by underscores for 64-bit.  */
#  ifndef __MINGW_USE_UNDERSCORE_PREFIX
     /* As we have to support older gcc version, which are using underscores
      as symbol prefix for x64, we have to check here for the user label
      prefix defined by gcc. */
#    ifdef __USER_LABEL_PREFIX__
#      pragma push_macro ("_")
#      undef _
#      define _ 1
#      if (__USER_LABEL_PREFIX__ + 0) != 0
#        define __MINGW_USE_UNDERSCORE_PREFIX 1
#      else
#        define __MINGW_USE_UNDERSCORE_PREFIX 0
#      endif
#      undef _
#      pragma pop_macro ("_")
#    else /* ! __USER_LABEL_PREFIX__ */
#      define __MINGW_USE_UNDERSCORE_PREFIX 0
#    endif /* __USER_LABEL_PREFIX__ */
#  endif
#else /* ! ifndef _X86_ */
   /* For x86 we have always to prefix by underscore.  */
#  undef __MINGW_USE_UNDERSCORE_PREFIX
#  define __MINGW_USE_UNDERSCORE_PREFIX 1
#endif /* ifndef _X86_ */
```
Adding this to bindgen.h resolves this:
``` h
#define __MINGW_USE_UNDERSCORE_PREFIX 0
```

**Comment by ctaggart** [comment]
fixed by #22
